test/jit/test_unsupported_ops.py

```diff
@@ -101,7 +102,7 @@ def tensordot():
         with self.assertRaisesRegex(Exception, "Argument dims_self"):
             torch.jit.script(tensordot)
 
-
+    @unittest.skipIf(not torch._C.has_lapack, "PyTorch compiled without Lapack")
     def test_init_ops(self):
```

Comment:
When did you run into this?

Reply:
This test always fails for me since I don't have lapack installed (see PR description for specific failure)